HW04/aleksei_zhuchkov/lib/notification.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Notification
+  attr_reader :title, :description
+
+  def initialize(title = 'None', description = 'None')
+    @title = title
+    @description = description
+  end
+
+  def get_notifications(notifications)
+    return 'None notification' if notifications.empty?
+
+    result = []
+    notifications.each { |el| result << [el.title, el.description] }
+    puts result
```

Comment:
```suggestion
   result
```

Reply:
I did it. Pls check https://github.com/labs-ruby/homeworks-2022/pull/96/commits/dae1efbc03eaff7f656807528fa5be337be720f5